Open_Foam/Domain.ipynb

```diff
@@ -52,23 +45,29 @@
     "\n",
-    "The vertices are automatically labeled by blockMesh by their order. The first vertex would be labeled as 0, the second vertex would be 1 and this numbering process continues up to the last vertex. The next step would be defining the two blocks based on the vertices. This is done by calling the vertices by their respective labels.\n",
+    "The above-defined vertices are automatically labeled numerically by `blockMesh` in the order that are input into the `blockMesh` file. First input vertex would be labeled with number 0, the second vertex would be labeled with number 1 and this labeling process continues up to the last input vertex. These labeling process is visualized in Fig. 3.\n",
     "\n",
-    "<pre>blocks\n",
-    "(\n",
-    "    hex (0 1 2 3 6 7 8 9) (25 200 1) simpleGrading (1 80 1)\n",
-    "    hex (1 4 5 2 7 10 11 8) (25 200 1) simpleGrading (1 80 1)\n",
-    ");</pre>\n",
+    "<img src=\"./Images/3labels.png\" width=\"500\">\n",
+    "Fig. 3. Labels of the vertices forming the blocks of the domain.\n",
+    "\n",
+    "At this stage that the skeleton of the CFD domain's geometry is defined the two blocks are created by \"connecting\" vertices together and simultaneously define the meshing process of them using the `blocks` command line in the `blockMesh` file. Each block is defined using one `hex` command. The inputs of the `hex` command are a list of vertex labels, described above, a 3D vector defining the number of mesh elements in each direction, and the type and list of cell expansion ratio in each direction as shown below. In this case study the number of mesh is chosen to be 100 and 200 in angular and radial direction respectively. Furthermore, the total expansion ratio in the radial direction is set to 80 distributed using `simpleGrading` algorithm.\n",
     "\n",
-    "The block definitions are contained in a list named blocks. Each block definition is a compound entry consisting of a list of vertex labels whose order is described in the previous section, a vector giving the number of cells required in each direction, the type and list of cell expansion ratio in each direction. Here the number of cells is chosen to be 100 and 200 in angular and radial direction respectively. Also the total expansion ratio in the radial direction is chosen to be 80.\n",
+    "```C++\n",
+    "blocks\n",
+    "(\n",
+    "    hex (0 1 2 3 6 7 8 9) (100 200 1) simpleGrading (1 80 1)\n",
+    "    hex (1 4 5 2 7 10 11 8) (100 200 1) simpleGrading (1 80 1)\n",
+    ");\n",
+    "```\n",
     "\n",
```

Comment:
Actually this is the default order in the blockMeshDict file. First the vertices are defined. In the next step blocks with respective mesh counts on each edges are defined. In some cases there are no curved edges. In the cases which curved edges exist user have to specify it in the next section. The order is same in all OpenFOAM official tutorials. So i think it is better to keep it that way, but also changing it won't effect the execution of the command.